src/ModelContextProtocol/Server/AIFunctionMcpServerPrompt.cs

```diff
@@ -144,7 +150,7 @@ private static AIFunctionFactoryOptions CreateAIFunctionFactoryOptions(
                 {
                     Name = param.Name,
                     Description = param.Value.TryGetProperty("description", out JsonElement description) ? description.GetString() : null,
-                    Required = param.Value.TryGetProperty("required", out JsonElement required) && required.GetBoolean(),
```

Comment:
Was this just broken before?

Reply:
Seems like it, it started failing once I started using complex types.